src/utils.py

```diff
@@ -55,152 +55,37 @@ def get_body(url: str, random_seed: float = None) -> str:
 
     encoding_mapper = encoding_mappings.EncodingMapper()
 
-    if 'theguardian.com' in url:
-
-        if "article-body-commercial-selector css-79elbk" in str(response.content):
-            # get content of article (as html tag)
-            article_content = soup.find(class_="article-body-commercial-selector css-79elbk")
-        elif "content__article-body from-content-api js-article__body" in str(response.content):
-
+    # List of URLs for news sites with their respective 'main body of text' HTML tags
+    news_sites = {
+
+        'theguardian.com': 'content__article-body from-content-api js-article__body',
+        'bbc.co.uk': 'story-body__inner',
+        'nytimes.com': 'meteredContent css-1r7ky0e',
+        'news.sky.com': 'sdc-site-layout-wrap site-wrap site-wrap-padding',
+        'metro.co.uk': 'article-body',
+        'huffingtonpost.co.uk': 'page__content__row row--no-border',
+        'inews.co.uk': 'article-padding article-content',
+        'telegraph.co.uk': 'articleBodyText section',
+        'mirror.co.uk': 'article-body',
+        'independent.co.uk': 'main-content'
+
+    }
+
+    for key in news_sites:
+        if key in url:
```

Comment:
We need an `else ` to this `if`, it looks like now people are not going to be able to go to bad websites through our tool, which is good. However if they have not put in a website we support then the code will do nothing, or break. We should raise an exception here which we can handle elsewhere, something like a `WebsiteNotSupported` exception, then we can deal with it elsewhere (maybe in the flask web code and then inform the user we don't support the requested website. In fact it may be better to put these as an enumeration in a separate script so they can be used easily by the web code to tell the user what websites we do actually support.

Reply:
I'm on to this (I think.)